client/src/store/reducer/surveyDetailReducer.js

```diff
@@ -27,6 +34,12 @@ const surveyDetailReducer = (state = initalState, action) => {
       return { ...state, successfulPublish: payload };
     case 'SET_SUCCESSFUL_CLOSE':
       return { ...state, successfulClose: payload };
+
+    case 'TOGGLE_OPEN_SLACK_MODAL':
+      return { ...state, openSlackModal: true };
```

Comment:
why is it called toggle if it always sets it to true. wouldn't a toggle function switch the value

Reply:
it's set to flase and clicking the button toggles it to true